backend/src/packages/chats/chats.ts

```diff
@@ -12,8 +15,14 @@ const chatRepository = new ChatRepository(ChatModel, UserToChatModel);
 const chatService = new ChatService({
   chatRepository,
   chatMessageService,
+  openAiService: openAi,
```

Comment:
```suggestion
  openAiService,
```
Let's use the same naming format for all